packages/botonic-react/src/webchat/messages-reducer.ts

```diff
@@ -71,6 +102,11 @@ function updateMessageReducer(
         ],
       }
     }
+
+    const unreadMessages = state.messagesComponents.filter(
+      messageComponent => messageComponent.props.isUnread
+    ).length
```

Comment:
Suggestion: Wouldn't it be better to rename this variable to `numUnreadMessages`? At first I thought it contained all the unread messages, but I've seen the `.length` at the end of the filter and I've seen then that it contains the number of unread messages.

Reply:
Applied